templates/ingredient/ingredient.js

```diff
@@ -88,8 +88,8 @@ function updateFixedHeader($productHeader) {
 }
 
 export default async function decorate(doc) {
-  const { productId, productName } = getUrlParams();
-  const [region, locale] = getRegionLocale();
+  const { productName } = getUrlParams();
+  const [, locale] = getRegionLocale();
```

Comment:
we would need to define region here since it is being consumed to build for url.